invenio/modules/search/static/js/search/search.js

```diff
@@ -18,293 +18,240 @@
-                    if ($(this).data('facet').findByKey(event.key).length === 0) {
-                        resetKey = true
-                    }
-                } catch(err) {
-                    resetKey = true
-                }
-                if (resetKey) {
-                    $('[data-facet="reset-key"]')
-                    .filter('[data-facet-key="'+event.key+'"]')
-                    .addClass('text-info')
-                }
-
-            }).on('added', function(event) {
-
-                var type = (event.op == '+')?'info':'error',
-                    other_type = (event.op == '-')?'info':'error'
-
-                $('[data-facet-key="'+event.key+'"]')
-                .filter('[data-facet-value="'+event.value+'"]')
-                .each(function(e) {
-                    if (event.op != '!') {
-                        $(this)
-                        .removeClass('text-'+other_type)
-                        .addClass('text-'+type)
-                        .css('font-weight', 'bold')
-                    }
-                })
-                .filter('.expandable')
-                .each(function() {
-
-                    var $el_clicked = $(this),
-                        options = facet_list.data('facet').options
-
-                    $el_clicked.removeClass('expandable')
-
-                    if (event.key === 'collection') {
-                        $(this).parent().find('ul').remove()
-                        var facet = {url: options.url_map[event.key],
-                            facet: event.key},
-                            data = {parent: event.value},
-                            $ul = $('<ul class="context list-unstyled"><ul>').clone().appendTo($el_clicked.parent())
-
-                        var data_facet = facet_list.data('facet')
-                        if (data_facet.find('!', event.key, event.value).length === 0) {
-                            data_facet.filter.push(['!', event.key, event.value])
-                        }
-                        data_facet.createFacetBox(facet, $ul, data)
-                    }
-                })
-
-                $('[data-facet="reset-key"]')
-                .filter('[data-facet-key="'+event.key+'"]')
-                .removeClass('text-info')
-                .addClass('muted')
-
-
-            }).on('exists', function(event) {
-                alert(facets.labels.alreadyin);
-            });
-
-            // Rebuild facet filter on hash change.
-            $(window).bind('hashchange', function() {
-                var hash = decodeURIComponent(document.location.hash),
-                    hash_filter = hash.substr(1),
-                    filter;
-
-                try {
-                    filter = $.parseJSON(hash_filter)
-                } catch (exc) {
-                    console.exception(exc)
-                }
+define([
+        'jquery',
+        'vendors/URIjs/src/URI',
+        'flight/lib/component',
+        './facets_filter',
```

Comment:
maybe better to include abs path like https://github.com/inveniosoftware/invenio/pull/3400/files#diff-adcf435606f6def2c24216a197668bf3R25 ?